pages/api/hello.ts

```diff
@@ -5,6 +5,6 @@ type Data = {
   name: string
 }
 
-export default function handler(req: NextApiRequest, res: NextApiResponse<Data>) {
+export function handler(req: NextApiRequest, res: NextApiResponse<Data>) {
```

Comment:
can we simply remove the whole file now? We don't need API routes

Reply:
Right, I will do it.